### ISSUE TITLE
Getting "Parse error" since vscode 1.52.0

### ISSUE THREAD

**Issue by storm1er** [wongjn/vscode-php-sniffer#45]
**Describe the bug**
Better give some screenshot to explain well what happens:

Brace on a new line
![gnome-shell-screenshot-XP4ZV0](https://user-images.githubusercontent.com/9821946/103269586-6e663680-49b6-11eb-863c-2035967903a9.png)

Brace on the same line
![gnome-shell-screenshot-KX89V0](https://user-images.githubusercontent.com/9821946/103269662-9190e600-49b6-11eb-91d2-837631a700ff.png)

This cause side effect like this
![gnome-shell-screenshot-WOXJW0](https://user-images.githubusercontent.com/9821946/103269719-aff6e180-49b6-11eb-8c0f-456733f0f264.png)

**To Reproduce**
1. Create a basic PHP class
2. Trigger sniff/lint
3. See the lint error

**Expected behavior**
Braces should be correctly detected.

**Environment**
 - OS: Ubuntu 18.04 LTS
 - VSCode version: 1.52.1
 - PHP CLI version: 7.3.24
 - squizlabs/PHP_CodeSniffer version: 2.9.2 project scoped

**Extension settings**
```json
{
    "phpSniffer.autoDetect": true,
    "phpSniffer.standard": "/.../project-workspace/build/config/phpcs/Calleo3",
    "phpSniffer.executablesFolder": "/.../project-workspace/bin/"
}
```

**Additional context**
Was working fine on vscode 1.51.0

**Comment by storm1er** [comment]
Damned, the error was raised by something else, see https://github.com/squizlabs/PHP_CodeSniffer/issues/316
